src/idtr.cpp

```diff
@@ -489,21 +504,41 @@ WaitHandleBase *_idtr_copy_reshape(SHARPY::DTypeId sharpytype,
     }
   }
 
+  int64_t oStride = std::accumulate(oDataStridesPtr, oDataStridesPtr + oNDims,
+                                    1, std::multiplies<int64_t>());
```

Comment:
I don't think this is what you want. For oNDims>1 this will never be 1.
You probably want to use `SHARPY::is_contiguous`.

Reply:
Thanks! Done!